### PR TITLE
Updated extension with alloy.js v2.0.0

### PR DESCRIPTION
**Pull request by ninaceban**
## Description

Updated extension with alloy.js v2.0.0
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All tests pass and I've made any necessary test changes.
- [ ] I've updated the schema in extension.json or no changes are necessary.

### CODE REVIEW

File: src/lib/runAlloy.js [hunk 1]
```diff
@@ -6033,7 +6033,7 @@ governing permissions and limitations under the License.
 
 // The value will be swapped with the proper version at build time
 // see rollupPluginReplaceVersion.js
-      var libraryVersion = "1.0.0";
```
Comment (Aaronius) [inline src/lib/runAlloy.js:6036]: Nothing else changed?

Comment (ninaceban) [reply src/lib/runAlloy.js:6036]: yes, comparing with master branch. I have updated the runAlloy.js file in my last PR related to remove syncIdentity command.

Comment (ninaceban) [reply src/lib/runAlloy.js:6036]: Here I just bumped the versions

Comment (Aaronius) [reply src/lib/runAlloy.js:6036]: Okay, cool. Looks good then.

**Pull request merged**